Formula/scrollkeeper.rb

```diff
@@ -19,6 +19,9 @@ class Scrollkeeper < Formula
     sha256 x86_64_linux:   "81a0061c55b44717079288661e0ecaac1b22d6a72c72b89e98237eb51c099539"
   end
 
+  # Last release on 2003-12-06 and no upstream activity.
+  deprecate! date: "2023-02-16", because: :unmaintained
```

Comment:
https://repology.org/project/scrollkeeper/history - dropped by FreeBSD, Gentoo, and Nix. Not available in popular distros like Debian, Fedora, Arch, etc.

Mostly replaced by `rarian`. Although that formula hasn't had a new release in a long time, there is upstream activity https://gitlab.freedesktop.org/rarian/rarian/-/tags with rc tag in last few days.